roles/lib_openshift/src/lib/base.py

```diff
@@ -22,6 +22,29 @@ def __init__(self,
         self.verbose = verbose
         self.kubeconfig = Utils.create_tmpfile_copy(kubeconfig)
         self.all_namespaces = all_namespaces
+        self.oc_binary = self._locate_oc_binary()
+
+    @staticmethod
+    def _locate_oc_binary():
+        ''' Find and return oc binary file '''
+        # https://github.com/openshift/openshift-ansible/issues/3410
+        # oc can be in /usr/local/bin in some cases, but that may not
+        # be in $PATH due to ansible/sudo
+        paths = os.environ.get("PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]
+        paths = list(set(paths))
+
+        oc_binary = 'oc'
+        if six.PY2:
```

Comment:
@rhcarvalho we *could*. 

Ideally, we'd be able to use a library, but there doesn't seem to be one that we can just use without some work.

RHEL/CentOS/Fedora have python-which, but it is old and doesn't appear to be maintained. 

Fedora has [python2-backports-shutil_which](https://koji.fedoraproject.org/koji/packageinfo?packageID=23667), which would give us a backported version of `shutil.which`, but we would need to do the CentOS/RHEL packaging.

Thoughts on going down this path until we could get python2-backports-shutil_which backported?

The code implementation is small, but I'd prefer not to be in the business of managing a which implementation: https://github.com/minrk/backports.shutil_which